### DISCUSSION TITLE
Can I add a custom discord command to send a particular message over to discord?

### DISCUSSION THREAD

**Discussion by can-ylmaz** [DiscordSRV/DiscordSRV#1612] [f. Questions] [answered]
My server changes its IP whenever it restarts due to how I set it up (ngrok). I'll extract the current ip from ngrok and I want an /ip or !ip command on discord which will make the DiscordSRV bot print out the current IP of the server assigned by ngrok. I don't want to add another bot for this so is there any way I can pull it off with DiscordSRV?

**Comment by thewilloftheshadow** [comment]
You'd likely need to use a placeholder or find a way to update the config automatically with your start script

**Reply by can-ylmaz** [reply]
I know that, I was asking if I could make such a command/auto-response within discordSRV so I don't need a second bot

**Reply by thewilloftheshadow** [reply, marked as answer]
Ah, then yes! Look at the CannedResponses line in the config, that should be what you're looking for

**Reply by can-ylmaz** [reply]
Hi again, I set discordSRV up today, it has no console channel but only a chat channel which is properly configured but whenever anyone types anything to the channel it detects as a command and gives an error DM. Do you happen to know the fix?

**Reply by thewilloftheshadow** [reply]
https://discordsrv.com/discord and open a ticket
